presto-spi/src/test/java/io/prestosql/spi/block/TestLongArrayBlockEncoding.java

```diff
@@ -13,48 +13,30 @@
  */
 package io.prestosql.spi.block;
 
-import io.airlift.slice.DynamicSliceOutput;
-import org.testng.annotations.Test;
+import io.prestosql.spi.type.Type;
+
+import java.util.Random;
 
-import static io.prestosql.spi.block.BlockTestUtils.assertBlockEquals;
 import static io.prestosql.spi.type.BigintType.BIGINT;
 
 public class TestLongArrayBlockEncoding
+        extends BaseBlockEncodingTest<Long>
 {
-    private final BlockEncodingSerde blockEncodingSerde = new TestingBlockEncodingSerde();
-
-    @Test
-    public void testRoundTripNoNull()
+    @Override
+    protected void write(BlockBuilder blockBuilder, Long value)
```

Comment:
nit: orders of methods should match order in `BaseBlockEncodingTest`

Reply:
I'll try to fix that before the automation kicks in

Reply:
done